<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook ms9">3.004 <span class="authors"></span></div>
</div></header>


<footer class="navbar navbar-default navbar-fixed-bottom mcs-psy-nav" >
  <div class="container-fluid text-right h4">
    <em>ψᵟ</em>
  </div>
</footer>


### Given   
A closed, rigid tank contains 1.25 kg of water, initially a two phase liquid–vapor mixture at $T_1$ = 60 °C. Heat transfer occurs until the tank contains only saturated vapor at $T_2$ = 120 °C.  


### Find  
 1. $P_1$ [kPa]  
 1. $Q$ [kJ]    



### Get Organized
Hide the children! Beautiful diagram lurketh:

![R-134a Saturation Pressure Table](../figures/ms9,3-004.png)

These things you should notice right away:
1. Our system is in a **rigid container** ⋮ expect **constant volume** (isochoric/isometric) process.  
1. Our mass is constant. Constant mass and constant volume $\ce{->}$ **constant specific volume** process.   
1. We can resolve State 2 with $T_2$ (given) and $v_2 = v_{g@T_2}$ (saturated vapor, given).    
1. We can resolve State 1 with $T_1$ (given) and $v_1 = v_2$ (constant specific volume, remember?).

Always, always, **always**: Find the state you can resolve. Relate that state to the states you need via the processes that connect them. Ride that train for the rest of your thermodynamic life. It will bring you Home every time.

#### Find $\normalsize \boldsymbol{P_1}$
When we resolve State 1, we will have $P_1$. No problem.

#### Find $\normalsize \boldsymbol{Q}$
We must solve the Energy Equation to calculate $Q$. We may as well get in the habit now of addressing the mass balance, then address the energy balance. 

**Mass**:  
Trivial. Closed System.

**Energy:**  
1. We have an isometric process, so no boundary work.
1. We have no other forms of work present.
1. We are told nothing about system velocity or elevation, so we assume $\Delta \ce{KE} = \Delta \ce{PE} = 0$.  
1. We'll assume the heat flows into the system.

So our energy equation is as follows:  

\begin{array}{ccccc}
  E_{in} &-& E_{out} &=& \Delta E_{sys} \\
  Q_{in} &&&=& U_{2} - U_{1} \\{}
\end{array}

$$ Q_{in} = m\left(u_2 - u_1  \right) $$


We'll need $u_1$ and $u_2.$
 

### Attack

#### Resolve State 2 to get $\normalsize \boldsymbol{v_2, u_2}$
    
$\quad$ State 2 ⋮ $\left. \left\{ \begin{array}{} 
                     T_2 = \ce{120°C} \\
                     v_2 = v_{g@T_2}
                    \end{array} \quad \right| \quad 
                    \begin{array}{l} 
                     v_2 = 0.8919\,\ce{m^3/kg} \\ 
                     u_2 = 2529.3\,\ce{kJ/kg}
                    \end{array} \quad \right.$ 
                    
                    
#### Resolve State 1 to get $\normalsize \boldsymbol{u_1}$
    
$\quad$ State 1 ⋮ $ \left\{ \begin{array}{} 
                     T_1 = \ce{60°C} \\
                     v_1 = v_2 = 0.8919 \,\ce{m^3/kg}
                    \end{array} \quad \right| \quad v_{f@60\ce{°C}} = 0.0010172\,\ce{m^3/kg} < v_1 < v_{g@60\ce{°C}} = 7.671\,\ce{m^3/kg} \ \vdots$ saturated mix.



<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 

It's saturated. **Immediately**, we know

\begin{gather*} \normalsize
     P_1 = P_\text{sat@60°C} = 0.1994\,\ce{bar} \\{}
\end{gather*}

</div></div>

 Using quality to get $u_1:$
 
\begin{align*}
 x &= \frac{v_1 - v_\text{f@60°C}}{v_\text{g@60°C} - v_\text{f@60°C}} &&= \frac{u_1 - u_\text{f@60°C}}{u_\text{g@60°C} - u_\text{f@60°C}} \\[9pt]
   &= \frac{\big(0.8919 - 0.0010172\big)\ce{m^3/kg}}{\big(7.671 - 0.0010172\big)\ce{m^3/kg}} &&= \frac{u_1 - 251.11\,\ce{kJ/kg}}{\big(2456.6 - 251.11\big)\ce{kJ/kg}}
\end{align*}  

                    

In [1]:
# ⋮⋮⋮⋮⋮ Julia initializations ⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮

using Unitful     

In [2]:
v₁ = 0.8919u"m^3/kg"
u₂ = 2529.3u"kJ/kg"
m = 1.25u"kg"

vf_60°C = 0.0010172u"m^3/kg"
vg_60°C = 7.671u"m^3/kg"

uf_60°C = 251.11u"kJ/kg"
ug_60°C = 2456.6u"kJ/kg"

x₁ = (v₁ - vf_60°C)/(vg_60°C - vf_60°C)

u₁ = uf_60°C + x₁*(ug_60°C - uf_60°C)
@show u₁

Q_in = m * (u₂ - u₁)
@show Q_in;

u₁ = 507.2817748013724 kJ kg⁻¹
Q_in = 2527.5227814982845 kJ


<div class="btn-success">&nbsp;<div class="badge"></div><i class="fa fa-spin fa-fw fa-cog"></i><strong> Finis</strong></div>
<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 
\begin{gather*} \normalsize
    P_1 = 19.94\,\ce{kPa}, \quad Q = 2527.52\,\ce{kJ}, \textbf{IN}
\end{gather*}
</div></div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>